pyaerocom/helpers.py

```diff
@@ -1704,18 +1726,22 @@ def get_max_period_range(periods):
 
 
 def make_dummy_cube(
-    var_name: str, start_yr: int = 2000, stop_yr: int = 2020, freq: str = "daily", dtype=float
+    var_name: str,
+    start_yr: int = 2000,
+    stop_yr: int = 2020,
+    freq: str = "daily",
+    dtype=float,
 ) -> iris.cube.Cube:
-    startstr = f"days since {start_yr}-01-01 00:00"
+    startstr = f"{TS_TYPE_TO_FREQ_NAME[freq]} since {start_yr}-01-01 00:00"
 
     if freq not in TS_TYPE_TO_PANDAS_FREQ.keys():
         raise ValueError(f"{freq} not a recognized frequency")
 
     start_str = f"{start_yr}-01-01 00:00"
-    stop_str = f"{stop_yr}-12-31 00:00"
+    stop_str = f"{int(stop_yr)+1}-01-01 00:00"
```

Comment:
Might be wrong but `pd.date_range` is inclusive on both ends by default, so I think this includes one more element than we want? https://pandas.pydata.org/docs/reference/api/pandas.date_range.html

Reply:
True, this was added to bypass a bug (maybe) with reading the Cube. But I think I've fixed it, so this will be removed